comparison-configuration/description.md

```diff
@@ -1 +1,3 @@
-This is a comparison of all available Internet of Things plattforms. It is based on the publication [A gap analysis of Internet-of-Things platforms, Computer Communications (2016)](http://dx.doi.org/10.1016/j.comcom.2016.03.015) by J. Mineraud et al..
\ No newline at end of file
+This is a comparison of all available Internet of Things plattforms. It is based on the publication [A gap analysis of Internet-of-Things platforms, Computer Communications 
```

Comment:
Can you do a better line break here?
Maybe put `[ A graph analysis....](...)` in a separate line?
Also for the next reference?